selfdrive/ui/tests/cycle_offroad_alerts.py

```diff
@@ -11,8 +10,20 @@
 if __name__ == "__main__":
   params = Params()
 
-  with open(os.path.join(BASEDIR, "selfdrive/controls/lib/alerts_offroad.json")) as f:
-    offroad_alerts = json.load(f)
+  offroad_alerts = (
+    'Offroad_TemperatureTooHigh',
+    'Offroad_ConnectivityNeeded',
+    'Offroad_UpdateFailed',
+    'Offroad_InvalidTime',
+    'Offroad_UnofficialHardware',
+    'Offroad_StorageMissing',
+    'Offroad_BadNvme',
+    'Offroad_CarUnrecognized',
+    'Offroad_ConnectivityNeededPrompt',
+    'Offroad_NoFirmware',
+    'Offroad_IsTakingSnapshot',
+    'Offroad_NeosUpdate',
+  )
```

Comment:
Is there any way to list params and then just take the ones with this prefix?

Reply:
There is no way to get a list of all keys. how about keep it as it is? we can write test in C++ or add a method to return all keys in Params late if necessary

Reply:
add a new method get_all_keys in https://github.com/commaai/openpilot/pull/25779